01week/rockPaperScissors.js

```diff
@@ -43,6 +63,18 @@ if (typeof describe === 'function') {
       assert.equal(rockPaperScissors('Paper', 'SCISSORS'), "Hand two wins!");
       assert.equal(rockPaperScissors('rock ', 'sCiSsOrs'), "Hand one wins!");
     });
+    //Test for all possible scenarios in which "Hand one wins!".
+    it('should Test for all possible scenarios in which "Hand one wins!"', () => {
+      assert.equal(rockPaperScissors('rock', 'scissors'), "Hand one wins!");
+      assert.equal(rockPaperScissors('paper', 'rock'), "Hand one wins!");
+      assert.equal(rockPaperScissors('scissors', 'paper'), "Hand one wins!");
+    });
+    //Test for all possible scenarios in which "Hand two wins!".
+    it('should Test for all possible scenarios in which "Hand two wins!"', () => {
+      assert.equal(rockPaperScissors('paper', 'scissors'), "Hand two wins!");
+      assert.equal(rockPaperScissors('scissors', 'rock'), "Hand two wins!");
+      assert.equal(rockPaperScissors('rock', 'paper'), "Hand two wins!");
+    });
   });
```

Comment:
@JacobGonz1993 
You are missing the test where you make sure that the input is a valid entry: https://javascript.austincodingacademy.com/_book/02Week/02DayClass.html#instructions